Reflections/R2 - April 23.md

```diff
@@ -0,0 +1,18 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+Over the last week, the largest coal that I have accomplished is the implementation of my modular structure building system. The player is now able to select the structure they would like to build from the structures backstage menu, and then place the structure that they have chosen on any non occupied hex tile in the game map. When they attempt to do this, the system does the math to make sure that the x is unoccupied, and that the player has enough materials in their inventory necessary to build the structure once it's built and placed onto the game map it will function in the same way that the example structures from the proof-of-concept worked. I've also built this system in a way that adding in additional structures will be a relatively simple process.
```

Comment:
Truly, fantastic progress on structures. Seeing the working demo in class-time, I can really tell you tackled this goal head-on to some great results.